addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1034,9 +1034,9 @@ def _to_store(self, store: Store, fields):
         store.add_records_fields(self, fields)
 
     # User methods
+
     @api.model
-    @api.returns("self", lambda channels: Store(channels).get_result())
-    def channel_get(self, partners_to, pin=True, force_open=False):
+    def _channel_get_or_create_chat(self, partners_to, pin=True, force_open=False):
```

Comment:
Prefix `_channel` is not necessary, especially on channel model.
```suggestion
    def _get_or_create_chat(self, partners_to, pin=True, force_open=False):
```